### ISSUE TITLE
OpenNESS deployment got failed. [Errno -1] repomd.xml signature could not be verified for kubernetes

### ISSUE THREAD

**Issue by amr-mokhtar** [open-ness/converged-edge-experience-kits#107]
During a smoke test today of the OpenNESS 21.03 release, we have observed an issue which is affecting the deployment of OpenNESS that seems to be related to the [Kubernetes issue #60134](https://github.com/kubernetes/kubernetes/issues/60134)
```
Error occurred. Will retry after 5 seconds. You can find details below:
Failure talking to yum: failure: repodata/repomd.xml from kubernetes: [Errno 256] No more mirrors to try.
https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64/repodata/repomd.xml: [Errno -1] repomd.xml signature could not be verified for Kubernetes
```

A quick fix is to disable `repo_gpgcheck` in file `converged-edge-experience-kits/roles/kubernetes/common/tasks/main.yml`:
```diff
    gpgkey: "{{ _kubernetes_repository_key }}"
    gpgcheck: yes
    enabled: true
-    repo_gpgcheck: true
  become: yes
  register: result
  retries: "{{ number_of_retries }}"
```

This workaround is currently under verification by our nightly builds with a view to providing a patch release once we are sure it does not cause any regressions.

**Comment by ravicorning** [comment]
Sure that work around seems to work.

**Comment by amr-mokhtar** [comment]
Closing as this has been fixed by tag `openness-21.03.01`
